client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/ContactDateCard/InteractionCard/InteractionCard.tsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import { format } from 'date-fns';
+import { Card, Collapse, IconButton, Typography, Grid, Divider } from '@material-ui/core';
+import { KeyboardArrowDown, KeyboardArrowLeft, Edit, Delete } from '@material-ui/icons';
+
+import Interaction from 'models/Interaction';
+
+import useStyle from './InteractionCardStyles';
+
+const InteractionCard: React.FC<Props> = (props: Props) => {
```

Comment:
Me and Maya are calling what you are displaying here an 'InteractionEvent' so i'd change the name to 'InteractionEventCard'